docs/sources/developers/plugins/working-with-data-frames.md

```diff
@@ -1,20 +1,20 @@
 ---
-title: Working with data frames
+title: Work with data frames
 ---
 
-# Working with data frames
+# Work with data frames
 
-The data frame is a columnar data structure which allows efficient querying of large amounts of data. Since data frames are a central concept when developing plugins for Grafana, in this guide we'll look at some ways you can use them.
+The [data frame](data-frames.md) is a columnar data structure that allows for efficient querying of large amounts of data. Since data frames are a central concept when developing plugins for Grafana, in this guide we'll look at some ways you can use them.
```

Comment:
```suggestion
The [data frame]({{< relref "data-frames" >}}) is a columnar data structure that allows for efficient querying of large amounts of data. Since data frames are a central concept when developing plugins for Grafana, in this guide, we'll look at some ways you can use them.
```